eslint.config.mjs

```
@@ -1,7 +1,7 @@
-import {dirname} from 'path';
-import {fileURLToPath} from 'url';
+import { dirname } from 'path';
```

ESLint

[error] There should be no space after '{'.

(object-curly-spacing)